tests/unit/test_client.py

```diff
@@ -7775,6 +7775,43 @@ def test_load_table_from_json_w_invalid_job_config(self):
         err_msg = str(exc.value)
         assert "Expected an instance of LoadJobConfig" in err_msg
 
+    def test_load_table_from_json_unicode_emoji_data_case(self):
+        from google.cloud.bigquery.client import _DEFAULT_NUM_RETRIES
+        from google.cloud.bigquery import job
+
+        client = self._make_client()
+
+        json_row = {"emoji": "\U0001F3E6"}
```

Comment:
Can you make the emoji a constant, so that we can use it directly in an assertion below?  E.g.:
```suggestion
        emoji = "\U0001F3E6"
        json_row = {"emoji": emoji}
```